back-end/validation/register.js

```diff
@@ -22,6 +23,10 @@ module.exports = function validateRegisterInput(data) {
     errors.email = "Email is invalid";
   }
 
+  // Username checks
+  if (Validator.isEmpty(data.username)) {
```

Comment:
In addition to checking if the username is empty, I think we'll also want to make sure there aren't any special characters (like spaces)